From bb44b85f17918762731405f90760521c6caa7d14 Mon Sep 17 00:00:00 2001
From: Andrew Abbott <andrew@mirx.dev>
Date: Sat, 14 May 2022 09:04:18 +1000
Subject: [PATCH] lightbar: Enable BLUE_PULSING

---
 common/lightbar.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/common/lightbar.c b/common/lightbar.c
index b2901b664..7adac4d69 100644
--- a/common/lightbar.c
+++ b/common/lightbar.c
@@ -6,6 +6,8 @@
  * LED controls.
  */
 
+#define BLUE_PULSING
+
 #ifdef LIGHTBAR_SIMULATION
 #include "simulation.h"
 #else
-- 
2.36.0

